android/app/src/main/java/com/go_social/MainApplication.java

```diff
@@ -1,53 +1,71 @@
 package com.go_social;
```

Comment:
I think it is safe to remove all these commented lines in this java file.

Reply:
I kept them for two reasons: firstly, to preserve a record of the changes made since they belonged to the previous version; secondly, the new version of React Native automatically links new packages, that's why I commented them out. However, I kept them for future reference, and when the app is ready for production, I will remove the extra comments if I don't need them anymore.

Reply:
No need to worry about tracking the changes as git is already doing it for us. Otherthing is we always should write code focussing production so that we no longer have extra work to do.

Reply:
Thanks, i will remove all those comments.